### ISSUE TITLE
With Nano S it does not work 

### ISSUE THREAD

**Issue by diustudio** [hussy-io/truffle-ledger-provider#5]
I tried make a clean new project, and config the truffle.js as readme, it still no work.

Just run and nothing outputed.

**Comment by diustudio** [comment]
![image](https://user-images.githubusercontent.com/40688388/42199224-e2cf83d8-7ebf-11e8-8ad6-bb39f3eea745.png)

**Comment by alejoamiras** [comment]
Same error.

**Comment by stevenleeg** [comment]
I can add some more information here, as I appear to be running into a similar issue as everyone here. Here's the output when I run `truffle migrate`:

```
$ truffle migrate --network ropsten --reset --verbose-rpc
   > {
   >   "jsonrpc": "2.0",
   >   "id": 1,
   >   "method": "eth_accounts",
   >   "params": []
   > }```

... and then the commande exits. It appears there's some difficulty with the provider and the `eth_accounts` RPC call which is causing things to fail.

For reference, here is my configuration:
```js
const LedgerWalletProvider = require('truffle-ledger-provider');
const INFURA_KEY = '...';

const ledgerOptions = {};

const provider = new LedgerWalletProvider(
  ledgerOptions,
  `https://ropsten.infura.io/v3/${INFURA_KEY}`,
);

module.exports = {
  networks: {
    ropsten: {
      provider: provider,
      network_id: 3,
      gas: 4600000,
    },

    development: {
      host: 'localhost',
      port: 8545,
      network_id: '*',
    },
  },
};
```

I've also ensured that my Ledger is connected to my computer and configured with data support _enabled_ and browser support _disabled_.

Any ideas as to what could be the problem here?

**Comment by tomericco** [comment]
I moved to use `@0xproject` subprovider, combined with `web3-provider-engine` -- and now everything works with my Ledger Nano. Take a look at it here: https://github.com/0xProject/0x-monorepo/blob/development/packages/subproviders/src/subproviders/ledger.ts

**Comment by Ykid** [comment]
I reported a bug to ledgerjs and it turns out to be a bug in the ethereum app. They fixed this bug in v1.1.4 and I tested it, it works fine. Guys can follow the code example in the issue to do deployment.

https://github.com/LedgerHQ/ledgerjs/issues/205

**Comment by andreafspeziale** [comment]
@Ykid Any way to get it work?
The mentioned issue is closed.

I've tried to place a couple of console.logs and seems that `sendAsync` method is broken.
@petertulala Any hint?

Thanks 4 ur work!
